### ISSUE TITLE
Lighthouse a11y score of 55

### ISSUE THREAD

**Issue by argyleink** [GoogleChromeLabs/tooling.report#214]
We should have a higher accessibility score than `55`. 

Here's what Lighthouse says we need to resolve to fix our score:
- [ ] some contrast ratios need fixed
- [ ] html element needs a `lang` attribute
- [ ] images need `alt` attributes
- [ ] Github FAB needs some labels for screenreaders

Fixing these will raise the `74` score on SEO as well
